Makefile

```diff
@@ -83,18 +83,18 @@ endif
 
 all: manager
 
+GINKGO_VERSION ?= v2.8.3
+GINKGO_TIMEOUT ?= 2h
+GINKGO_RUN ?= go run github.com/onsi/ginkgo/v2/ginkgo@$(GINKGO_VERSION) -v -coverprofile cover.out -timeout $(GINKGO_TIMEOUT)
+
 .PHONY: unittest
-unittest: generate lint fmt vet manifests
-	go test -v -coverprofile cover.out $(SRC_PATHS_TESTS)
-	cd api && go test -v ./...
+unittest: generate fmt vet manifests
```

Comment:
Should depend on `gingko` target too? No more linting?

Reply:
Yup, bad rebase thanks for catching that.